src/qibo/gates/abstract.py

```diff
@@ -391,7 +391,7 @@ def apply_density_matrix(self, backend, state, nqubits):
         return backend.apply_gate_density_matrix(self, state, nqubits)
 
     def apply_clifford(self, backend, state, nqubits):
-        return backend.apply_gate_clifford(self, state[:-1], nqubits)
+        return backend.apply_gate_clifford(self, state, nqubits)
```

Comment:
Why did you remove this?

Reply:
Because index `-1` now is not the scratch row anymore, but rather the 8 bit integer that also include the scratch but it's not limited to it.

Reply:
Ok, it definitely makes sense. I was just confused thinking that it was cropping the phases column, not the scratch row.